notebook.ipynb

```diff
@@ -11,11 +11,11 @@
     "\n",
```

Comment:
<p>Thank you :)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/smith-kyle/deep-learning-with-python/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>